Teacher.rb

```diff
@@ -0,0 +1,10 @@
+require './person'
+
+class Teacher 
```

Comment:
- [ ] Same issue here, the `Teacher` class should inherit from the `person` class :+1:

Reply:
done

Reply:
Hello @Richez003, Kindly also add the `attr_accessor :name, :age` and `attr_reader :id` to your Person class so as to be able to access those variables :+1: